tests/system/provisioning/agentless_cluster/roles/master-role/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
       - policycoreutils
       - automake
       - autoconf
-      - libtool
+      - libtool 
```

Comment:
Could you explain why this appears here, please?

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2955/commits/546dcf98325b3a94eb580acc997fa4cc44463938